### DISCUSSION TITLE
Sonoff - Automate your conventional heating - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#53] [Comments]
# Sonoff - Automate your conventional heating - Haade.fr

Transform your NSPanel Pro into a boiler contactor by directly integrating the Sonoff RE5VC micro module and control everything using the Ewelink application

https://haade.fr/en/blog/replace-your-conventional-heating-system-with-modules-from-the-sonoff-range

**Comment by zalwandri78** [comment]
What are the two communication protocols mentioned in the article, and why is mixing them a concern? regard <a href=" https://jakarta.telkomuniversity.ac.id/menyelami-dunia-botnet-ancaman-tersembunyi-di-era-digital/ "> Tel U </a>
